src/_locales/nb/messages.json

```diff
@@ -23,6 +26,15 @@
 	"appDescription": {
 		"message": "En universell sync-tjeneste for Trakt.tv."
 	},
+	"autoSync": {
+		"message": "Automatic Sync"
+	},
+	"autoSyncDescription": {
+		"message": "The number of days determines how often the automatic sync will happen for the streaming service. For the automatic sync to be activated, you must sync manually in the history page of the streaming service first, so that the last sync date can be saved.\n\nIt's also important to note that when loading the history of the streaming service, it will only load the history until the last item that was automatically synced. To reset this behavior, you have to clear the last sync date for the streaming service."
```

Comment:
Antall dager avgjør hvor regelmessig automatisk synkronisering vil forekomme for den angitte strømmetjenesten. For å aktivere automatisk synkronisering må du først synkronisere manuelt i historikken en gang, slik at en synkroniseringsdato kan lagres.\n\nNB! Når du laster inn historikken til strømmetjenesten så vil den kun laste inn historikken opp til datoen på siste automatiske synkronisering.